lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +236,28 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        tags = article_bs.find_all('a', class_='news-listing__item-link')
+        url = ''
+        for element in tags:
+            url = element['href']
+        return url
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
+        urls = []
+        while len(urls) < self.config.get_num_articles():
+            for url in self.get_search_urls():
+                response = make_request(url, self.config)
+
+                if not response.ok:
+                    continue
+
+                soup = BeautifulSoup(response.text, 'lxml')
+                urls.append(self._extract_url(soup))
```

Comment:
just get tags here and not in extract_url

tags = article_bs.find_all('a', class_='news-listing__item-link')

than call extract on ech tag